pkg/services/secrets/manager/manager_test.go

```diff
@@ -300,23 +294,7 @@ func TestSecretsService_Run(t *testing.T) {
 		// Then, once the ticker has been triggered,
 		// the cleanup process should have happened,
 		// therefore the cache should be empty.
-		require.Len(t, svc.dataKeyCache, 0)
-	})
-
-	t.Run("should update data key expiry after every use", func(t *testing.T) {
```

Comment:
I intentionally removed that behavior cause we've explicitly been requested to add support for the opposite behavior (not TTL renewals) and I see no strong reasons to keep support for both strategies (code would become more complex, etc with no clear benefits).

In terms of performance, I think the overhead of an extra network request (to decrypt the data encryption key against the external key vault) every fifteen minutes (or so) should be more than acceptable.